src/main/java/study3/RandomGenerator.java

```diff
@@ -0,0 +1,18 @@
+package study3;
+
+import java.util.Random;
+
+/*
+ * random값 생성하는 클래스
+ * author : choi
+ */
+public class RandomGenerator {
+
+	private static int randomrange = 10;
+
+	public int RandomGenerate() {
```

Comment:
메소드의 첫 글자는 소문자로 시작해주세요.
```suggestion
    public int randomGenerate() {